src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -738,7 +738,7 @@ protected function checkCategories($hookData, Github $github, Logger $logger, $p
 	 */
 	protected function checkFilesAndAssignCategory($files)
 	{
-		$addCategories = [];
+		$addCategories = array();
```

Comment:
I'm not sure i understand what you mean? What do you expect to be there? The last thing i know is that [] and array() is the same? I'm wrong with it?